core/color.cpp

```diff
@@ -428,8 +415,7 @@ Color Color::named(const String &p_name) {
 	if (color) {
 		return color->value();
 	} else {
-		ERR_EXPLAIN("Invalid Color Name: " + p_name);
-		ERR_FAIL_V(Color());
+		ERR_FAIL_V_MSG(Color(), "Invalid Color Name: " + p_name + ".");
```

Comment:
The `if` / `else` could now be simplified as:
```cpp
ERR_FAIL_NULL_V_MSG(color, Color(), "Invalid Color Name: " + p_name + ".");

return color->value();
```
but that's not mandatory to do in this PR, the current code is fine too.